.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Can you remove this prettier config changes? Would like to keep this PR self-contained to only have the blob url changes.

Reply:
:+1: Sure, I will update it.